Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
You should clone https://github.com/pierrre/gotestcover to `$GOPATH/github.com/pierrre/gotestcover`, then add https://github.com/ruflin/gotestcover as a secondary Git remote.